addons/binding/org.openhab.binding.knx/ESH-INF/config/channelConfig.xml

```diff
@@ -4,46 +4,97 @@
 	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
         http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
-	<!-- Dimmer -->
-	<config-description uri="channel-type:knx:dimmer">
-		<parameter name="switch" type="text">
-			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to toggle the dimmer on or off</description>
-			<required>false</required>
-		</parameter>
-		<parameter name="position" type="text">
-			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to set the absolute position of the dimmer</description>
-			<required>false</required>
-		</parameter>
-		<parameter name="increaseDecrease" type="text">
-			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to increase or decrease the dimmer</description>
-			<required>false</required>
-		</parameter>
-	</config-description>
-	<config-description uri="channel-type:knx:dimmer-control">
-		<parameter name="switch" type="text">
-			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to toggle the dimmer on or off</description>
-			<required>false</required>
-		</parameter>
-		<parameter name="position" type="text">
-			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to set the absolute position of the dimmer</description>
-			<required>false</required>
-		</parameter>
-		<parameter name="increaseDecrease" type="text">
-			<label>Address</label>
-			<description>The group address(es) in Group Address Notation to increase or decrease the dimmer</description>
-			<required>false</required>
-		</parameter>
-		<parameter name="frequency" type="integer">
-			<label>Frequency</label>
-			<description>Increase/decrease send frequency if dimming should be handled by the binding (in ms) - set to 0 if the KNX device sends them repeatedly itself</description>
-			<default>0</default>
-		</parameter>
-	</config-description>
+    <!-- Dimmer -->
```

Comment:
Could you apply the auto-formatter? It seems you replaced the tabs by spaces in here.